foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -43,13 +43,12 @@ This becomes more important later on when we learn about manipulating HTML with
 
 Back in the `index.html` file, let's add the `<html>` element by typing out its opening and closing tags, like so:
 
-~~~html
+```html
 <!DOCTYPE html>
-<html lang="en">
-</html>
-~~~
+<html lang="en"></html>
+```
 
-#### What is the `lang` attribute? 
+#### What is the lang attribute?
```

Comment:
```suggestion
#### What Is the Lang Attribute?
```

Reply:
Of course! My apologies, I didn't realize there was more to do:)